Pack3/games/AwShirt/content/AwShirtSlogans.jet

```diff
@@ -992,7 +992,7 @@
         },
         {
             "id": 40567,
-            "suggestion": "[KE RALPOLENG] Today sucks",
+            "suggestion": "Wiya fìtrr fe' leru",
```

Comment:
I think maybe something with "pak" would be fitting, but I don't know the best way to say it